src/main/java/io/lettuce/core/AbstractRedisAsyncCommands.java

```diff
@@ -389,6 +389,11 @@ public RedisFuture<String> clientTracking(TrackingArgs args) {
         return dispatch(commandBuilder.clientTracking(args));
     }
 
+    @Override
+    public RedisFuture<List<Object>> clientTrackinginfo() {
```

Comment:
The return type should be Map<String, String> to match what we have for other similar commands.

Reply:
> The return type should be Map<String, String> to match what we have for other similar commands.

Which similar commands?